ci-jobs/pr-validation-build.yml

```diff
@@ -11,22 +11,19 @@ stages:
   jobs:
   - template: ./templates/xcuitest-e2e-template.yml
     parameters:
-      name: iPhoneX_15_2
-      iosVersion: 15.2
-      xcodeVersion: 13.2
-      deviceName: "iPhone Simulator"
```

Comment:
How do you decide 'generic' simulator? https://github.com/appium/appium-xcuitest-driver/blob/master/lib/ios-generic-simulators.js
Apparently, the CI env no longer has `iPhone X` for iOS 16.2 https://dev.azure.com/AppiumCI/Appium%20CI/_build/results?buildId=24430&view=logs&j=19e62f86-4150-54d4-d8db-ae5426c0dd0e&t=edcc93af-4c73-59ad-5b41-a879e5dae5b2&l=116

Reply:
there are two ways to do it:
either you hardcode the device model and it gets outdated after a while when Apple stops supporting it or fetch the model dynamically by parsing the output of `xcrun simctl list` in the CI script

Reply:
I was just wondering if there is a policy to pick which device to use for tests. If you don't mind, I'm going to hard code iPhone 12 & 13

Reply:
We don't have it right now. Some test cases potentially depend on the device resolution etc, but as XCUITest driver' we don't have such a policy. (in WDA, we'd like to run tests in various combinations of Xcode/OS versions)

Reply:
I see, thank you for clarifying!